proto/attestation/reference_value.proto

```diff
@@ -89,6 +89,17 @@ message StringReferenceValue {
   repeated string values = 1;
 }
 
+message Regex {
+  string value = 1;
+}
+
+message RegexReferenceValue {
```

Comment:
I was thinking that we would have a StringReferenceValue, which may be either skip, or a literal string, or a regex. Not that the reference value itself would be about regex.

And the field name would just be `kernel_cmd_line` (instead of `kernel_cmd_line_regex`).

Anyways this can go in the bucket of future improvements.